Add learning goals to some registry and data tutorials (#2895)
viamrobotics/docs: docs/tutorials/configure/pet-photographer.md

no_list: true
---

<!-- LEARNING GOALS

> After following this tutorial, you will understand how to control sync parameters, so you can build filters that allow you to filter the data you are collecting from your machines.

Reviewer: Nit: to avoid `so`:

```suggestion
After following this tutorial, you will understand how to control sync parameters and build filters that allow you to filter the data you are collecting from your machines.
```
